io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties

```diff
@@ -14,9 +14,9 @@ io.worker.task-keepalive=Specify the number of milliseconds to keep non-core tas
 io.buffer-pool=Defines buffer pool
 io.buffer-pool.add=Adds new buffer pool
 io.buffer-pool.remove=Removes buffer pool
-io.buffer-pool.buffers-per-slice=How many buffers per slice, if not set optimal value is calculated based on available RAM resources in your system.
-io.buffer-pool.buffer-size=The size of each buffer slice in bytes, if not set optimal value is calculated based on available RAM resources in your system.
-io.buffer-pool.direct-buffers=Does the buffer pool use direct buffers, some platforms don't support direct buffers
+io.buffer-pool.buffers-per-slice=Maximum number of buffers
+io.buffer-pool.buffer-size=Size of the all buffer at initialization
```

Comment:
@kabir yes, it is a bit confusing indeed. So when I look at the code, I saw that the NIO susbsys allocated one large buffer, at initialization, that one can then either use fully or sliced. The buffer-size value is obviously this value.

However, it can be somewhat confusing with the NIO documentation which also has concept of slice and buffer. This is what I try (and fail) to describe accurately here. Would this be a better description:

"Total size of the buffer handled by the subsystem" ?